### PR TITLE
Check that default value isn't an execution error

### PR DESCRIPTION
**Pull request by esjee**

### CODE REVIEW

**Comment by Willianvdv** [comment]
@esjee this fixes #798, right?

**Comment by esjee** [comment]
I believe so, yes!

File: lib/graphql/query/literal_input.rb [hunk 1]
```diff
@@ -77,6 +77,10 @@ def self.from_arguments(ast_arguments, argument_defns, variables)
             # `context` isn't present when pre-calculating defaults
             if context
               value = arg_defn.prepare(value, context)
+              if value.is_a?(GraphQL::ExecutionError)
```
Comment (Willianvdv) [inline lib/graphql/query/literal_input.rb]: Nice bug 😄

**Comment by rmosolgo** [comment]
:+1: Awesome, thanks for taking care of this!

**Pull request merged**